src/whylogs/core/statistics/numbertracker.py

```diff
@@ -153,10 +157,10 @@ def from_protobuf(message: NumbersMessage):
         number_tracker : NumberTracker
         """
         theta = None
-        if message.theta is not None and len(message.theta) > 0:
-            theta = ThetaSketch.deserialize(message.theta)
-        elif message.compact_theta is not None and len(message.compact_theta) > 0:
+        if message.compact_theta is not None and len(message.compact_theta) > 0:
```

Comment:
why did the order change ?

Reply:
`.theta` is no longer supported by datasketches. Only compact serialization

Reply:
👍